osmomath/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/osmosis-labs/osmosis/osmomath
 go 1.20
 
 require (
-	cosmossdk.io/math v1.1.3-rc.0
+	cosmossdk.io/math v1.1.3-rc.1
```

Comment:
Just double checking, do we want this bump?

Reply:
I think so, since we are utilizing it already in our main go.mod

Reply:
Plus this is only getting introduced in a major, so seems low risk